packages/mapping.yaml

```diff
@@ -24,6 +24,15 @@ maintainers:
 - name: "Philip Nienhuis"
   contact: "prnienhuis@users.sf.net"
 versions:
+- id: "1.4.3"
+  date: "2025-05-02"
+  sha256: "2188b24639c7087840172e6047f75cb8a598d758120ae6e0810ce6300a869a1b"
+  url: "https://downloads.sourceforge.net/project/octave/Octave%20Forge%20Packages/Individual%20Package%20Releases/mapping-1.4.3.tar.gz"
+  depends:
+  - "octave (>= 5.2.0)"
+  - "io (>= 2.2.7)"
+  - "geometry (>= 4.0.0)"
+  - "pkg"
```

Comment:
``` suggestion
  - "pkg"
 ubuntu2204:
  - "libgdal-dev"
```

That would indicate to the CI to install the GDAL package (for building and running with more enabled features).